hq/app/views/main.scala.html

```diff
@@ -22,10 +22,9 @@
 
         @views.html.header()
 
-        @pageHeader
-
         <main>
-        @content
+          @pageHeader
```

Comment:
Moving pageheader into one of the html sections, so it is not alone for Christmas 🎄🎁

(also, it should be a child of either `header`, `main` or `footer`)

Reply:
Great spot!